lib/wickedbase.pm

```diff
@@ -124,14 +124,15 @@ sub create_tunnel_with_commands {
 }
 
 sub setup_bridge {
-    my ($self, $config, $dummy) = @_;
+    my ($self, $config, $dummy, $command) = @_;
     my $local_ip = $self->get_ip(no_mask => 1, is_wicked_ref => 0, type => 'host');
+    $command = 'ifup' if (!$command);
```

Comment:
I think better to just not have default value , it will be more clear for caller .

Reply:
what do you mean? that I shall update all the places where we call this function with'ifup' ?